resilience4j-spring/src/main/java/io/github/resilience4j/utils/SpringCloudContextMissingOnClasspathCondition.java

```diff
@@ -0,0 +1,18 @@
+package io.github.resilience4j.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+public class SpringCloudContextMissingOnClasspathCondition implements Condition {
+
+    private static final Logger logger = LoggerFactory.getLogger(SpringCloudContextMissingOnClasspathCondition.class);
+    private static final String CLASS_TO_CHECK = "org.springframework.cloud.context.scope.refresh.RefreshScope";
+    @Override
+    public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+        return AspectUtil.checkClassIfNotFound(context, CLASS_TO_CHECK, (e) -> logger.info("RefreshScope are activated because AspectJ is on the classpath."));
```

Comment:
I think we don't have to log anything, if Spring Cloud Config is on the classpath.